modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json

```diff
@@ -40,6 +40,7 @@
     "Undo": "Undo",
     "Update": "Update",
     "YourComment": "Your comment",
-    "YourReply": "Your reply"
+    "YourReply": "Your reply",
+    "CmsKit:Medias:0001": "'{Name}' is not a valid media name."
```

Comment:
`Permission:MediaDescriptorManagement:Create` and `Permission:MediaDescriptorManagement:Delete` keys are missing

Reply:
Also `Media` is already Plural, using `Media` instead of `Medias` is better.